components/SearchForJobs.tsx

```diff
@@ -1,10 +1,69 @@
-import * as React from "react";
-import { Text, View } from "react-native";
+import React, { useState } from "react";
+import { SafeAreaView, TextInput, Button, StyleSheet } from "react-native";
+import axios from "axios";
 
 export const SearchForJobs = () => {
+  const [jobs, onChangeJobs] = useState("");
+  const [location, onChangeLocation] = useState("");
+  const [data, setData] = useState([]);
+
+  const handleSearch = () => {
+    let formattedJobs = "";
+    for (const letter of jobs) {
```

Comment:
remove unnecessary formatting. url encoding takes care of the space for you

Reply:
Done